app/assets/javascripts/app/app.js

```diff
@@ -170,5 +170,6 @@ var app = {
 
 $(function() {
   app.initialize();
+  $("#update_profile_form").areYouSure();
```

Comment:
I'm not sure this the best location for this specific call. I think https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/app/pages/settings.js is hooked up to the profile edit page?